bower.json

```diff
@@ -31,22 +31,25 @@
   ],
   "dependencies": {
     "firebase": "^2.1.0",
-    "ember-data": "^1.13.0"
+    "ember-data": ">=1.13.6"
   },
   "devDependencies": {
-    "ember": "1.13.3",
+    "ember": "2.0.2",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
     "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
-    "ember-mocha": "0.8.0",
-    "ember-test-helpers": "0.5.1",
-    "ember-resolver": "0.1.18",
+    "ember-mocha": "0.8.2",
+    "ember-test-helpers": "0.5.9",
+    "ember-resolver": "^0.1.18",
     "jquery": "2.1.4",
     "loader.js": "ember-cli/loader.js#3.2.1",
     "moment": "2.10.3",
     "blueimp-md5": "1.1.0",
     "normalize-css": "3.0.3",
-    "sinon": "http://sinonjs.org/releases/sinon-1.15.4.js",
+    "sinon": "http://sinonjs.org/releases/sinon-1.16.1.js",
     "mockfirebase": "0.11.0"
+  },
+  "resolutions": {
```

Comment:
Yes, it can be removed, as it was only to avoid getting the question for which ember version bower wants to install when running ```bower install```. I'll update the PR without the resolution part.